content/docs/reference/glossary.md

```diff
@@ -242,10 +262,18 @@ Server Name Indication. A TLS protocol extension that allows a client or browser
 
 A feature of the Neon Console that enables running queries on a Neon database. The SQL Editor also enables saving queries, viewing query history, and analyzing or explaining queries.
 
+### start_compute
+
+A Neon Control Plane operation that starts a compute endpoint when there is an event or action that requires compute resources. For example, connecting to a suspended compute endpoint initiates this operation. See [Operations](https://neon.tech/docs/manage/operations) for more information.
+
 ### Storage
 
 Where data is recorded and stored. Neon storage consists of Pageservers, which store hot data, and a cloud object store, such as Amazon S3, that stores cold data for cost optimization and durability.
 
+### suspend_compute
+
+A Neon Control Plane operation that suspends a compute endpoint after a period of inactivity. For information about how Neon manages compute resources, see Compute lifecycle. See [Operations](https://neon.tech/docs/manage/operations) for more information.
+
```

Comment:
We can reference this page too https://neon.tech/docs/introduction/compute-lifecycle

Reply:
Thank you. Added.